testing/unit_testing/src/2_ranks/laplace_1d.cpp

```diff
@@ -30,7 +30,7 @@ config = {
     "OutputWriter" : [
       {"format": "Paraview", "filename": "out4", "outputInterval": 1, "binary": False, "fixedFormat": True, "combineFiles": False, "onlyNodalValues": True},
       {"format": "PythonFile", "filename": "out4", "outputInterval": 1, "binary": False, "onlyNodalValues": True},
-      {"format": "HDF5", "filename": "out4", "outputInterval": 1, "combineFiles": True},
+      {"format": "HDF5", "filename": "out4", "outputInterval": 1, "combineFiles": False},
```

Comment:
Why did you have to change this to `False` for the test to pass?

Reply:
Idealy we test both the combinedFiles and without, not sure why isnt working in CI, it was working on HPC (also already on the pervious PR #50) (my target maschine, as opendihu does not work at all locally for me)